public/app/plugins/panel/canvas/editor/elementEditor.tsx

```diff
@@ -68,6 +68,7 @@ export function getElementEditor(opts: CanvasEditorOptions): NestedPanelOptions<
         currentOptions = {
           ...layer.getNewOptions(options),
           type: layer.id,
+          name: `Element ${Date.now()}.${Math.floor(Math.random() * 100)}`,
```

Comment:
In a new PR, lets resolve getting a unique name from the scene -- and having the scene keep a name > element Map (currently based on UID)